toolbox

```diff
@@ -638,6 +744,42 @@ case $op in
         $ls_containers && list_containers
         exit
         ;;
+    rm | rmi )
+        force=false
+        all=false
+        while has_prefix "$1" -; do
+            case $1 in
+                -f | --force )
+                    force=true
+                    ;;
+                -a | --all )
+                    all=true
+                    ;;
+                * )
+                    exit_if_unrecognized_option $1
+            esac
+            shift
+        done
+        if $all; then
+            id=""
+            exit_if_extra_operand $1
+        elif ! $all && [ "$1" = "" ]; then
+            echo "$0: missing argument for '$op'"
+            echo "Try '$0 --help' for more information."
+            exit 1
+        else
+            id=""
+            while [ "$1" != "" ]; do
+                id="$id $1"
```

Comment:
The variable with the list of IDs will have an extra preceding whitespace. We should  either avoid that in the first place or remove it after the loop to prevent weird behaviour inside the `remove_*` functions. Depending on exactly how you access the variable you either see the whitespace or you don't. This seems to matter when iterating over the IDs.

Shells are weird. It's better to be idiot-proof. :)

Reply:
Done